integrated_channels/integrated_channel/models.py

```diff
@@ -241,6 +242,23 @@ def clean(self):
                 }
             )
 
+    def fetch_linked_content_audits(self):
```

Comment:
important to note, this lives on every customer config and will grab all content transmission audits that are connected to a catalog associated with the config.